src/main/java/org/zalando/jackson/datatype/money/MonetaryAmountSerializer.java

```diff
@@ -16,6 +16,7 @@
 
     private final MonetaryAmountFormatFactory factory;
     private final FieldNames names;
+    private final boolean numberAsString;
```

Comment:
I initially implemented features the same way as jackson does, but then discarded it since I hope there will not be that many toggles at all